docs/manual/models/defining.md

```diff
@@ -0,0 +1 @@
+# Defining Models
```

Comment:
What's the plan for these placeholder files? Keep in or recreate once they're written?